.github/workflows/checkin.yml

```diff
@@ -0,0 +1,31 @@
+name: pull-request
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+      - uses: actions/checkout@v2
+      - name: Set up node 
+        uses: actions/setup-node@v1
+      - name: Install dependencies
+        run: |
+          id
+          whoami
+          ps -ef
+          pwd
+          sudo apt-get update -qq
```

Comment:
i know `-qq` is seen in a lot of runs out there, but tbh, i don't know why.  the GH actions output will collapse steps once they're done running, so the default amount of log output isn't really a problem.